src/utils/enum.py

```diff
@@ -850,6 +858,138 @@ def __str__(self) -> str:
         return self.short_name
 
 
+class TitleNorm(Enum):
+    WIM = auto()
+    WGM = auto()
+    IM = auto()
+    GM = auto()
+
+    @classmethod
+    def values(cls) -> Iterator['TitleNorm']:
+        yield cls.WIM
+        yield cls.WGM
+        yield cls.IM
+        yield cls.GM
+
+    def satisfies_gender_requirement(self, gender: PlayerGender) -> bool:
+        match self:
+            case TitleNorm.WIM | TitleNorm.WGM:
+                return gender == PlayerGender.FEMALE
+            case _:
+                return True
+
+    @property
+    def minimum_rating(self) -> int:
+        match self:
+            case TitleNorm.WIM:
+                return 1850
+            case TitleNorm.WGM:
+                return 2000
+            case TitleNorm.IM:
+                return 2050
+            case TitleNorm.GM:
+                return 2200
+            case _:
+                raise ValueError(f'Invalid title norm value: {self}')
+
+    @property
+    def minimum_average(self) -> int:
+        match self:
+            case TitleNorm.WIM:
+                return 2030
+            case TitleNorm.WGM:
+                return 2180
+            case TitleNorm.IM:
+                return 2230
+            case TitleNorm.GM:
+                return 2380
+            case _:
+                raise ValueError(f'Invalid title norm value: {self}')
+
+    @property
+    def minimum_performance(self) -> float:
+        match self:
+            case TitleNorm.WIM:
+                return 2250
+            case TitleNorm.WGM:
+                return 2400
+            case TitleNorm.IM:
+                return 2450
+            case TitleNorm.GM:
+                return 2600
+            case _:
+                raise ValueError(f'Invalid title norm value: {self}')
+
+    @staticmethod
+    def minimum_rounds(tournament: 'Tournament') -> int:
+        from data.pairings.variations import DoubleBergerRoundRobinVariation
+
+        if tournament.pairing_variation == DoubleBergerRoundRobinVariation():
+            return 10  # 1.4.5.f -> 6 players -> 10 rounds
+        return 9
+
+    @staticmethod
+    def minimum_score(
+        rounds: int, point_values: dict[Result, float] | None = None
+    ) -> float:
+        return Result.WIN.points(point_values) * 0.35 * rounds
```

Comment:
Probably doesn't need `point_values` any more
```suggestion
    @staticmethod
    def minimum_score(
        rounds: int
    ) -> float:
        return Result.WIN.points() * 0.35 * rounds
```